src/atomate2/vasp/jobs/adsorption.py

```diff
@@ -0,0 +1,437 @@
+"""Jobs used in the calculation of surface adsorption energy."""
+
+from __future__ import annotations
+
+import logging
+from collections import defaultdict
+from dataclasses import dataclass, field
+from typing import TYPE_CHECKING, Any
+
+from jobflow import Flow, Response, job
+from pymatgen.analysis.adsorption import AdsorbateSiteFinder
+from pymatgen.core import Element, Molecule, Structure
+from pymatgen.core.surface import SlabGenerator
+from pymatgen.io.vasp import Kpoints
+from pymatgen.io.vasp.sets import MPRelaxSet
+
+from atomate2.vasp.jobs.base import BaseVaspMaker
+
+# from atomate2.vasp.sets.core import RelaxSetGenerator, StaticSetGenerator
+
+if TYPE_CHECKING:
+    from atomate2.vasp.sets.base import VaspInputGenerator
+
+logger = logging.getLogger(__name__)
+
+
+@dataclass
+class BulkRelaxMaker(BaseVaspMaker):
+    """Maker for molecule relaxation.
+
+    Parameters
+    ----------
+    name: str
+        The name of the flow produced by the maker.
+    input_set_generator: VaspInputGenerator
+        The input set generator for the relaxation calculation.
+    """
+
+    name: str = "bulk_relax_maker__"
+    input_set_generator: VaspInputGenerator = field(
+        default_factory=lambda: MPRelaxSet(
+            user_kpoints_settings={"reciprocal_density": 200},
+            user_incar_settings={
+                "ISIF": 3,
+                "ENCUT": 700,
+                "GGA": "RP",
+                "EDIFF": 1e-5,
+                "LDAU": False,
+                "NSW": 300,
+                "NELM": 500,
+            },
+            auto_ispin=True,
+        )
+    )
+
+
+@dataclass
+class MolRelaxMaker(BaseVaspMaker):
+    """Maker for molecule relaxation.
+
+    Parameters
+    ----------
+    name: str
+        The name of the flow produced by the maker.
+    input_set_generator: VaspInputGenerator
+        The input set generator for the relaxation calculation.
+    """
+
+    name: str = "mol_relax_maker__"
+    input_set_generator: VaspInputGenerator = field(
+        default_factory=lambda: MPRelaxSet(
+            user_kpoints_settings=Kpoints.from_dict(
+                {
+                    "nkpoints": 0,
+                    "generation_style": "Gamma",
+                    "kpoints": [[3, 3, 3]],
```

Comment:
Is this class for the relaxation of a lone molecule? If so, what is the purpose of having `kpoints` of [3,3,3] here and in the static calculation?

Reply:
Hi @Andrew-S-Rosen yes it's a lone molecule. I will modify this kpoints setting as well. Thanks for pointing it out.